Lesson16/deniz/script.js

```diff
@@ -0,0 +1,231 @@
+const firstLetter = stringToCheck[0];
+  if (firstLetter === firstLetter.toUpperCase()) {
+    return true;
+  } else {
+    return false;
+  }
+
+}
+console.log(isFirstLetterUppercase("Televiision"));
+// 5. Write a function that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function checkNumber(num) {
+    if (num > 0) {
+        return "Positive";
+    } else if (num < 0) {
+        return "Negative";
+    } else {
+        return "Zero";
+    }
+}
+console.log("exercise 4:", checkNumber(5));
+console.log("exercise 4:", checkNumber(-2));
+console.log("exercise 4:", checkNumber(0));
+
+// 6. Write a function that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function isLeapYear(year) {
+    if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
+        return true;
+    } else {
+        return false;
+    }
+}
+console.log("exercise 6:", isLeapYear(2024));
+console.log("exercise 6:", isLeapYear(2025));
+
+
+// 7. Write a function that takes in a temperature and checks if it's above or below freezing using if else statement.
+function isAboveOrBelow(temperature) {
+	if (temperature < 0) {
+		return "Its frozen";
+	} else {
+		return "Its NOT frozen";
+	}
+}
+console.log("exercise 7:", isAboveOrBelow(-10));
+console.log("exercise 7:", isAboveOrBelow(10));
+
+// 8. Write a function that takes in a person's height and weight and checks if they are considered underweight, normal, overweight, or obese using if else statement.
+// Body Mass Index (BMI) is a person’s weight in kilograms (or pounds) divided by the square of height in meters (or feet). A high BMI can indicate high body fatness.
+//Formula: weight (kg) / [height (m)]2
+// Below 18.5	Underweight
+// 18.5 – 24.9	Healthy Weight
+// 25.0 – 29.9	Overweight
+// 30.0 and Above	Obesity
+
+// 9. Write a function that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
+function checkingAge(age) {
+	if (age <= 12) {
+		return "Child";
+	} else if (age >= 13 && age <= 19) {
+		return "Teenager";
+	} else if (age >= 20 && age <= 64) {
+		return "Adult";
+	} else if (age >= 65) {
+		return "Senior Citizen";
+	}
+}
+console.log("exercise 9:", checkingAge(12));
+console.log("exercise 9:", checkingAge(20));
+
+// 10. Write a function that takes age as a number and whether they are a student (yes or no). If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen". If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+function checkVoting(age, isStudent) {
```

Comment:
In cases like this where answer can be "yes" or "no", it's better to use a [boolean type ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean).